src/main/java/net/earthcomputer/clientcommands/command/arguments/TranslationQueryArgumentType.java

```diff
@@ -0,0 +1,139 @@
+package net.earthcomputer.clientcommands.command.arguments;
+
+import com.mojang.brigadier.StringReader;
+import com.mojang.brigadier.arguments.ArgumentType;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import com.mojang.brigadier.exceptions.DynamicCommandExceptionType;
+import com.mojang.brigadier.exceptions.SimpleCommandExceptionType;
+import com.mojang.brigadier.suggestion.Suggestions;
+import com.mojang.brigadier.suggestion.SuggestionsBuilder;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.command.CommandSource;
+import net.minecraft.text.Text;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Consumer;
+
+public class TranslationQueryArgumentType implements ArgumentType<TranslationQueryArgumentType.TranslationQuery> {
```

Comment:
Does this argument type represent a language as the examples imply? 'Query' is confusing.

Reply:
Oops, I forgot to change the examples. They should make it more clear.